### PR TITLE
Add new parameters to send via GTM

### PR DESCRIPTION
**Pull request by csubira**
https://github.com/CartoDB/product/issues/407

This PR adds:

- User job role (`String`)
- User in trial period (`Boolean`)

to GTM data sent to Google Analytics.

### CODE REVIEW

File: app/views/shared/_google_tag_manager.html.erb [hunk 1]
```diff
@@ -1,14 +1,17 @@
 <!-- Google Tag Manager -->
 <%# Tags for GTM are being included in VendorScripts templates for static pages as well.
    # So if you change anything here, please make sure that you change it there too. %>
-
+<% byebug %>
```
Comment (alonsogarciapablo) [inline app/views/shared/_google_tag_manager.html.erb]: Careful here ☝️

File: lib/assets/javascripts/dashboard/components/vendor-scripts/vendor-scripts-view.js [hunk 2]
```diff
@@ -35,7 +35,9 @@ module.exports = CoreView.extend({
         userName: this._userModel.get('username'),
         userId: this._userModel.get('id'),
         userAccountType: this._userModel.get('account_type'),
-        userCreatedAtInSeconds: Date.parse(this._userModel.get('created_at')) / 1000
+        userCreatedAtInSeconds: Date.parse(this._userModel.get('created_at')) / 1000,
+        userJobRole: this._userModel.get('job_role'),
+        userInTrialPeriod: this._userModel.get('show_trial_reminder')
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/dashboard/components/vendor-scripts/vendor-scripts-view.js]: Not sure if we'll need to convert this into a string...

File: lib/assets/javascripts/new-dashboard/core/trackers.js [hunk 3]
```diff
@@ -21,7 +21,9 @@ if (tagManagerId) {
   window.dataLayer.push({
     userId: store.state.user.id,
     userAccountType: store.state.user.account_type,
-    userSignUpDate: Date.parse(store.state.user.created_at) / 1000
+    userSignUpDate: Date.parse(store.state.user.created_at) / 1000,
+    userJobRole: store.state.user.job_role,
+    userInTrialPeriod: store.state.user.show_trial_reminder
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/new-dashboard/core/trackers.js]: Same here, perhaps we need to convert to string, but I'm not sure.

File: app/views/shared/_google_tag_manager.html.erb (hunk 1 above)
Comment (csubira) [reply app/views/shared/_google_tag_manager.html.erb]: 🤦‍♀

File: lib/assets/javascripts/new-dashboard/core/trackers.js (hunk 3 above)
Comment (csubira) [reply lib/assets/javascripts/new-dashboard/core/trackers.js]: Sure, changed already!

File: lib/assets/javascripts/dashboard/components/vendor-scripts/vendor-scripts-view.js (hunk 2 above)
Comment (csubira) [reply lib/assets/javascripts/dashboard/components/vendor-scripts/vendor-scripts-view.js]: Checked in staging that it's casted to string automatically! But good catch anyway!

**Comment by csubira** [comment]
@alonsogarciapablo Deployed in a dedicated instance, check it out [here](https://test-gtm.carto-staging.com/dashboard)

**Comment by alonsogarciapablo** [comment]
I missed this @cillas, sorry! Please go ahead and deploy it if you feel it's working 👌. Thank you!

**Comment by alonsogarciapablo** [comment]
Thank you @csubira! 👏

**Pull request merged**